docs/components/Layout.tsx

```diff
@@ -1,8 +1,7 @@
 import * as React from 'react';
-import { Link, Route } from 'react-router-dom';
-import { styled, transparentize, distance, colors } from '../../src';
-import { ScrollToTop } from './ScrollToTop';
+import { styled, transparentize, distance, colors, reStyled, Anchor } from '../../src';
 import { MobileMenu } from './MobileMenu';
+import { breakpoints } from '../../src/themes';
```

Comment:
Why not get them from the import above?

```ts
import { styled, transparentize, distance, colors, reStyled, Anchor } from '../../src';
```

The reason we use this import is for dogfooding reasons. It is also the thing that is exported / seen by users importing the lib. So if we use stuff here that can only be known by importing from subpaths then our kitchen sink / demo would not be so nice.